src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -87,6 +87,7 @@ public void testIsUserRegistered_false() {
   private void assertEquals(User expectedUser, User actualUser) {
     Assert.assertEquals(expectedUser.getId(), actualUser.getId());
     Assert.assertEquals(expectedUser.getName(), actualUser.getName());
+    Assert.assertEquals(expectedUser.getPassword(), actualUser.getPassword());
```

Comment:
Does the test still pass with this added? It looks to me like the code under test is just writing null for the returned user's password here.

Reply:
I think at this point it is writing null -- the projects have it split up so that password is just set to null in part1, and part2 is where we get password from Datastore

Reply:
Ahh, I see--this is testing the UserStore, which stores in-memory, so it's able to retrieve the passwords correctly, and that's what we're testing here.

The PersistentDataStore only returns null for the password, so that's why you aren't testing that until after the changes in the next section.

Sorry for the confusion!

Reply:
No worries!